tests/functional/spec/exit.spec.js

```diff
@@ -20,6 +20,6 @@ describe("Post submission exit", () => {
     $(CensusThankYouPage.exit()).click();
     browser.back();
     expect(browser.getUrl()).to.contain("submitted/thank-you");
-    expect($("body").getHTML()).to.contain("Your session has expired");
+    expect($("body").getHTML()).to.contain("Your census has been submitted");
```

Comment:
Probably needs the extra string ("This page is no longer available") as the thank you page could feasibly include "Your census has been submitted"